start.sh

```diff
@@ -21,11 +61,13 @@ if [ "$NIC" = "virtio" ]; then
     fi
 fi
 
-qemu-system-x86_64 -enable-kvm \
+if [ "$IMG" = "" ]; then IMG="$(ls -1t *.qcow2 | head -1)"; fi
+
+$QEMUSYS -enable-kvm \
     -drive "file=$IMG" \
     $CDIMAGE \
     -net nic,model=$NIC \
     -net user \
-    -m 1024M \
+    -m "$RAM"M \
```

Comment:
I'd make the M part of ``$RAM`` variable, that means you could pass ``-m 1G`` to the script, for less typing and being more obvious to people used to QEMU's ``-m`` argument.